### ISSUE TITLE
PageTitel not changing 

### ISSUE THREAD

**Issue by alorson** [robiso/wondercms#192]
Hi, i tested several (default) Template and PageTitel in Browser is not really changing. 
Is that correct? Why can i insert manually Page Titles in Backend, if this wont affect Frontend, even in default Template? 
Is this a BUG?

**Comment by StephanStanisic** [comment]
Hi Alorson,

Could you please explain a bit more on what you where trying to do? Are you changing the page title under current page > page title, general > menu, on the page itself? Where do you expect to see the title change?

If you change the title under current page > page title, you will see the title change in the title bar of the tab, and in the meta tag for search engines.
If you change the title under general > menu, you will see the title change in the menu, and in the url of the page (iirc).
If you change the title on the page itself, you will only see it change on the page itself.

Also, could you maybe send some screenshots of what you are doing and what results you get vs what you would expect? Some version info (installed plugins, version of WonderCMS and version of php) would be handy as well.

**Comment by robiso** [comment]
I believe @StephanStanisic answers is sufficient in explaining the page title vs website title.
TLDR:
Page title -> Current page title (shown only on specific pages).
Website title -> Actual website title (shown globally).

@alorson, please feel free to re-open this issue or continue commenting for additional support.
